test/helpers/generic_dao.js

```diff
@@ -24,8 +24,13 @@
 
 
 global.genericDAOTestBattery = function(daoFactory) {
+  var oldContext;
+
   describe('generic DAO tests', function() {
     beforeEach(function() {
+      oldContext = foam.__context__;
```

Comment:
Is this really part of this CL, or just a bad merge?  These things should be covered by testcontext.js now.